post-activation script doesn't source `/var/lib/generacy/wizard-credentials.env` — generacy-ai/generacy#589's env file is written and ignored

## Context

[generacy-ai/generacy#589](https://github.com/generacy-ai/generacy/issues/589) (merged in generacy-ai/generacy#591) added a wizard-env-writer to the control-plane: on \`bootstrap-complete\`, it unseals the cluster-local credstore and writes the credentials to \`/var/lib/generacy/wizard-credentials.env\` for bash scripts to source.

But nothing in cluster-base actually sources that file. [\`entrypoint-post-activation.sh\`](.devcontainer/generacy/scripts/entrypoint-post-activation.sh) calls \`setup-credentials.sh\` directly:

\`\`\`bash
# Step 1: configure git/gh credentials from the env vars the wizard delivered
bash /usr/local/bin/setup-credentials.sh
\`\`\`

\`setup-credentials.sh\` reads \`${GH_TOKEN:-}\` from its env. The env file the control-plane wrote is sitting unread on disk. \`GH_TOKEN\` is still empty, the original \`WARNING: GH_TOKEN not set\` is still logged, and post-activation still fails the same way it did before #589 shipped.

Verified on a live cluster:

\`\`\`
$ docker exec <orchestrator> ls -la /var/lib/generacy/wizard-credentials.env
-rw------- 1 node node 56 May 12 14:14 wizard-credentials.env

$ docker exec <orchestrator> env | grep GH_TOKEN
# (empty)

$ docker exec <orchestrator> cat /tmp/post-activation.log
[setup-credentials] WARNING: GH_TOKEN not set — git operations requiring auth will fail
\`\`\`

## Fix

In [\`entrypoint-post-activation.sh\`](.devcontainer/generacy/scripts/entrypoint-post-activation.sh), source the wizard env file before \`setup-credentials.sh\` runs:

\`\`\`diff
  log "Starting post-activation setup..."

+ # Source wizard-delivered credentials (written by control-plane's
+ # bootstrap-complete handler — see generacy-ai/generacy#589).
+ # set -a auto-exports each assigned variable so child processes inherit them.
+ WIZARD_CREDS="${WIZARD_CREDS_PATH:-/var/lib/generacy/wizard-credentials.env}"
+ if [ -f "$WIZARD_CREDS" ]; then
+     log "Sourcing wizard credentials from $WIZARD_CREDS"
+     set -a
+     # shellcheck disable=SC1090
+     source "$WIZARD_CREDS"
+     set +a
+ fi
+
  # Step 1: configure git/gh credentials from the env vars the wizard delivered
  bash /usr/local/bin/setup-credentials.sh
\`\`\`

Two notes:
- \`set -a\` / \`set +a\` is the right shape — \`set -a\` causes every assignment in the sourced file to be exported, so \`setup-credentials.sh\` (a subprocess) sees them. Without it, the vars are local to the sourcing script and the child gets nothing.
- The path is configurable via \`WIZARD_CREDS_PATH\` for testing / overrides; default matches the writer's default ([\`packages/control-plane/src/services/wizard-env-writer.ts:27\`](https://github.com/generacy-ai/generacy/blob/develop/packages/control-plane/src/services/wizard-env-writer.ts#L27)).

## Caveats

This alone doesn't make \`git clone\` succeed for GitHub App credentials. The writer currently sets \`GH_TOKEN=<value>\` where \`<value>\` is the wizard's stored installation metadata (\`{"installationId":..., "accountLogin":..., ...}\` — see [\`GitHubAppInstallStep.tsx:287-295\`](https://github.com/generacy-ai/generacy-cloud/blob/main/packages/web/src/components/clusters/bootstrap/steps/GitHubAppInstallStep.tsx#L287-L295)), not a usable installation access token. A companion generacy-cloud issue is needed to mint a real installation token at wizard time and include it in the credential payload — being filed alongside this one.

After **both** companions ship, \`git clone\` will actually authenticate. This issue alone is necessary but not sufficient.

## Test plan
- [ ] After cluster-base image rebuild + cluster start: \`docker exec <orchestrator> env\` shows \`GH_TOKEN\` populated post-bootstrap-complete (assuming the env file exists and is non-empty)
- [ ] \`/tmp/post-activation.log\` no longer contains \`WARNING: GH_TOKEN not set\` (because the env var is set even if its value is garbage per the cloud-side gap)
- [ ] Idempotent: running \`entrypoint-post-activation.sh\` twice works the same

## Related
- generacy-ai/generacy#589 (writer side — merged in #591)
- generacy-ai/generacy-cloud (to be filed) — token-minting companion
- generacy-ai/generacy#572 (cluster ↔ cloud contract umbrella)

--- a/.devcontainer/generacy/scripts/entrypoint-post-activation.sh
+++ b/.devcontainer/generacy/scripts/entrypoint-post-activation.sh
@@ -31,6 +31,18 @@ log() {
 
 log "Starting post-activation setup..."
 
+# Source wizard-delivered credentials (written by control-plane's
+# bootstrap-complete handler — see generacy-ai/generacy#589).
+# set -a auto-exports each assigned variable so child processes inherit them.
+WIZARD_CREDS="${WIZARD_CREDS_PATH:-/var/lib/generacy/wizard-credentials.env}"
+if [ -f "$WIZARD_CREDS" ]; then
+    log "Sourcing wizard credentials from $WIZARD_CREDS"
+    set -a
+    # shellcheck disable=SC1090
+    source "$WIZARD_CREDS"
+    set +a
+fi
+
 # Step 1: configure git/gh credentials from the env vars the wizard delivered
 bash /usr/local/bin/setup-credentials.sh
 

